main/src/ca/uwaterloo/flix/language/phase/Namer.scala

```diff
@@ -99,10 +99,10 @@ object Namer {
     case DesugaredAst.Declaration.Namespace(ident, usesAndImports0, decls0, loc) =>
       val ns = Name.NName(ident.sp1, ns0.idents :+ ident, ident.sp2)
       val usesAndImports = usesAndImports0.map(visitUseOrImport)
-      val declsVal = traverse(decls0)(visitDecl(_, ns))
+      val declsVal = ParOps.parMapSeq(decls0)(visitDecl(_, ns))
```

Comment:
I'm not sure that this buys us anything if we're already parallel over compilation units, especially since namespaces can be nested.

Reply:
But we are not? Maybe we  should be?

Reply:
Ah right, I misread the above. IMO we should just move the parallelism over the compilation units rather than namespaces. I'd guess the performance impact is the same, but not sure.

Reply:
I create a ticket.